lti_emailer/views.py

```diff
@@ -56,4 +59,11 @@ def lti_launch(request):
         "lti_emailer launched with params: %s",
         json.dumps(request.POST.dict(), indent=4)
     )
-    return redirect('mailing_list:index', request.POST['resource_link_id'])
+
+    view = 'mailing_list'
+    if has_lti_role(request, const.LEARNER):
```

Comment:
I think in practice a given user would only have one role, but yes, switching the logic makes sense for when they change their minds and want multiple roles per user. Changing now...